index.html

```diff
@@ -236,6 +233,8 @@ <h3>Skills</h3>
                         <button class="c-button" type="submit">Get in touch</button>
                     </form>
                 </article>
+            </section>
+            <section class="footer">
```

Comment:
👍 Great job on adding the footer section!

- [ ] However, instead of using a section tag, I recommend using the footer tag to semantically represent this section. This will improve the structure and accessibility of your webpage.

- [ ] Additionally, the gap in this section doesn't match the design. To achieve a closer match, please follow the design and adjust your code accordingly to ensure proper spacing.

- [ ] Lastly, I noticed there is an unwanted `>` element in the code. Please remove it to maintain clean and error-free code.
![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/55840999/cffb5fe6-6392-468b-903e-22ae5ef4c898)

By making these adjustments, your footer section will be more semantically correct and visually consistent with the design. Keep up the good work, and I'm excited to see your continued progress! 🚀🎉📝